packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -111,9 +128,17 @@
       height: var(--iui-size-3xs);
     }
   }
+
+  .iui-tabs-actions {
+    column-gap: var(--iui-size-xs);
+    margin-left: var(--iui-gap-between-inputs);
+  }
 }
 
 @mixin iui-tabs-vertical {
+  grid-template-areas: 'tabs tabs-content' 'tabs-actions tabs-content';
+  grid-template-columns: fit-content(1px) auto;
```

Comment:
what does this line do?

Reply:
This is the make the left column take only as much width as needed and the right column to take all the remaining space. But if there is a better way to achieve this, I could change it

Reply:
Without it, the width changes when switching tabs:

[Video6.webm](https://user-images.githubusercontent.com/45748283/210439288-d4f2762f-76a8-4d1e-8b39-489cdbb7d963.webm)

Reply:
i believe `grid-template-columns: auto 1fr` will achieve that. no need for `fit-content`

Reply:
Thanks, yes, I realized about `fr` later. So I changed it to `min-content 1fr` as mentioned in https://github.com/iTwin/iTwinUI/pull/860#discussion_r1061025070

Reply:
Looking at the docs for [`min-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/min-content), it seems like it should fail this situation and cause wrapping, but it seems to work fine, possibly because of `whitespace: nowrap` on the tab.
![](https://user-images.githubusercontent.com/9084735/210617466-24e82d60-2fad-4801-9a12-18092cebacf7.png)

[`auto`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns#:~:text=as%20a%20minimum.-,auto,-As%20a%20maximum) would still be more appropriate and resilient, but minor

Reply:
Thanks, changed